### ISSUE TITLE
Multiple H1 heading is defined on the home page: A11y_Maow_Home Page_Info and Relationships.

### ISSUE THREAD

**Issue by AnKushSingh05** [microsoft/moaw#109]
### GitHub Tags
#A11yTCS; #A11ySev2; #ChromiumEdge; #WCAG1.3.1; #Win11; #GH_Maow_Web_Mar24; #Maow; #Info and Relationships; #AILimited; #DesktopWeb; #A11yMAS;

### Actual:
Multiple H1 heading is defined on the home page.
### Expected:
Only one H1 heading should be defined on the page.
### Environment Details:
Application Name: Maow
#URL: https://moaw.dev/
OS: Windows 11 Enterprise 23H2 (OS build 22631.3296)
Microsoft Edge: Version 122.0.2365.92 (Official build) (64-bit)

### Repro Steps:
1. Hit the URL https://moaw.dev/
2. Moaw screen will get open.
3. TAB till header section.
4. Verify that Multiple H1 heading is defined on the home page or not.
### User Impact:
User not able to understand the current structure of the page if multiple H1 heading level tag present on the same page.

### Attachment:
![Presentation2](https://github.com/microsoft/moaw/assets/124244478/ead8faf3-6521-4f47-ae10-47e11864a02e)

**Comment by sinedied** [comment]
@AnKushSingh05 this one has been fixed, could you validate it?

**Comment by AnKushSingh05** [comment]
Verified the issue in below environment
https://moaw.dev/
Issue is fixed. Hence closing the bug

https://github.com/microsoft/moaw/assets/124244478/6ec6526f-b64a-4125-ad76-158ffce2a6db
